src/buffer/out/textBuffer.hpp

```diff
@@ -69,8 +69,6 @@ class TextBuffer final
                Microsoft::Console::Render::IRenderTarget& renderTarget);
     TextBuffer(const TextBuffer& a) = delete;
 
-    ~TextBuffer() = default;

```

Comment:
nit: don't we need one of these?

Reply:
Not sure how or why this got removed.

Reply:
Apparently fixing this caused static analysis to fail?